src/main/java/study/ResultView.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+class ResultView {
+    private final List<Set<LapResult>> results = new ArrayList<>();
+
+    public boolean isCommitted() {
+        return !results.isEmpty();
+    }
+
+    public void add(Set<LapResult> result) {
+        results.add(result);
+    }
+
+    public void report() {
+        print("실행결과");
+        for (int i = 0; i < results.size(); i++) {
+            print("\n");
```

Comment:
OS 마다 줄바꿈을 인식하는게 다르기 때문에 jdk 1.7부터 도입된 현재 OS에 맞는 줄바꿈을 얻어오도록 하면 좋을 것 같아요 :)

```suggestion
            print(System.lineSeparator());
```

Reply:
좋네요! 이런건 습관인데.. 😓  d9eacbcb433e3d864fc9af6a273110cebdff600c 에서 반영하였습니다.